src/components/footer.html

```
@@ -1,23 +1,22 @@
-<footer class="w-full premium-footer pb-6 pt-12 px-6 sm:px-8">
-  <div class="max-w-7xl mx-auto grid grid-cols-1 md:grid-cols-2 lg:grid-cols-5 gap-10 mb-12">
-    <!-- Branding Section -->
-    <div class="lg:col-span-2 flex flex-col gap-4">
-      <div class="flex items-center gap-3">
-        <div
-          class="w-10 h-10 rounded-xl bg-gradient-to-br from-primary to-primary-container flex items-center justify-center shadow-lg shadow-orange-500/20">
-          <span class="text-white font-extrabold text-xl font-headline">G</span>
+<footer class="premium-footer" aria-labelledby="footer-brand">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)